tests/unit/jobs/test_job_configuration_factory.py

```diff
@@ -22,7 +22,7 @@ def jade_data():
             {
                 "name": "australia",
                 "return_code": 1,
-                "status": "unfinished",
+                "status": "finished",
```

Comment:
Note the terminology change I made in the JobCompletionStatus enum. A job that fails does "finish"; it has a non-zero return code. Canceled and missing jobs do not finish.